projects.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use ProjectManagementApi\DatabaseConnection;
+use ProjectManagementApi\Hydrators\ProjectHydrator;
+use ProjectManagementApi\Response;
+
+header('Access-Control-Allow-Origin: *');
+header('Content-Type: application/json');
+
+
+try {
+    $db = new DatabaseConnection('mysql:host=db; dbname=project_manager', 'root', 'password');
+
+    $projectObjectsArray = ProjectHydrator::getAllProjects($db);
```

Comment:
same point here as my other comment - we don't need the word array in the name of the variable as long as it's plural

Reply:
Done!